server/mock_log_operation.go

```diff
@@ -6,8 +6,9 @@ package server
 
 import (
 	context "context"
-	gomock "github.com/golang/mock/gomock"
 	reflect "reflect"
+
+	gomock "github.com/golang/mock/gomock"
```

Comment:
Did `go generate` move this import?

Reply:
I guarantee it did.  For some of the directories, I had to run goimports for all the files.